server/routes/testutils/testData.ts

```diff
@@ -146,10 +147,20 @@ export default class TestData {
     ],
   }: Partial<CreateLocationGroupDto> = {}): CreateLocationGroupDto => ({ name, prisonId, locations })
 
+  static createCategoryGroupDto = ({
+    name = 'Wing A',
```

Comment:
```suggestion
    name = 'Category A (High Risk) prisoners',
```
Sounds more like a name of a location group rather than a category group. (Only test data - but might confuse us!)

Reply:
Updated